Platform/ArrowlakeBoardPkg/Script/StitchIfwiConfig_arls.py

```diff
@@ -147,8 +147,9 @@ def get_xml_change_list (platform, plt_params_list):
         # Path                                                                      | value |
         # =========================================================================================
         #   Region Order
-        ('./FlashSettings/BiosConfiguration/PchTopSwapOverride',                                    '512KB'),
-        ('./FlashSettings/BiosConfiguration/SocTopSwapOverride',                                    '512KB'),
+        ('./FlashSettings/BiosConfiguration/PchTopSwapOverride',                                    '4MB'),
```

Comment:
For Topswap settings, try using patch_xml_file() available in StitchIFwi for ARL-S.